lib/streamlit/config.py

```diff
@@ -185,11 +184,24 @@ def _delete_option(key):
         pass
 
 
+# Config Section: Mapbox #
```

Comment:
Move this section right before the S3 section. The order is actually important. It changes the order that things show up in when you run `streamlit config show`